gateway/ln-gateway/src/lib.rs

```diff
@@ -156,6 +156,17 @@ pub enum GatewayState {
     Disconnected,
 }
 
+impl From<GatewayState> for String {
```

Comment:
There is a already a trait for that: `impl fmt::Display for GatewayState` which will also implement the `ToString` trait giving you a `.to_string()`
See https://doc.rust-lang.org/std/fmt/trait.Display.html

Reply:
Fixed